client/www/pages/docs/instaml.md

```diff
@@ -391,8 +391,8 @@ db.tx.NAMESPACE_LABEL[ENTITY_IDENTIFIER].ACTION(ACTION_SPECIFIC_DATA)
 
 - `NAMESPACE_LABEL` refers to the namespace to commit (e.g. `goals`, `todos`)
 - `ENTITY_IDENTIFIER` is the id to look up in the namespace. This id must be a uuid and unique to the namespace. You can use the `id()` function to generate a uuid for convenience.
-- `ACTION` is one of `update`, `delete`, `link`, `unlink`
+- `ACTION` is one of `update`, `merge`, `delete`, `link`, `unlink`
 - `ACTION_SPECIFIC_DATA` depends on the action
-  - `update` takes in an object of information to commit
+  - `update` and `merge` takes in an object of information to commit
```

Comment:
`merge` could use its own description, e.g.

`merge` takes in an object to deep merge with the existing data